app/models/customer.py

```diff
@@ -0,0 +1,39 @@
+from flask import current_app
+from app import db
+from sqlalchemy import Table, Column, Integer, ForeignKey
+# from sqlalchemy.orm import relationship, back_populate
+# from app.models.video import Video
```

Comment:
we can get rid of these. Since we imported `db`, we can already use Column and the other imports like we did on lines 8-14

```suggestion

```